v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -46,12 +58,12 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
   void setJdbcDriverClassName(String driverClassName);
 
   @TemplateParameter.Text(
-      order = 3,
-      regexes = {"(^jdbc:mysql://.*|^gs://.*)"},
+      order = 4,
+      regexes = {"(^jdbc:mysql://.*|^jdbc:postgresql://.*|^gs://.*)"},
       groupName = "Source",
       description =
           "URL to connect to the source database host. It can be either of "
-              + "1. The JDBC connection URL - which must contain the host, port and source db name and can optionally contain properties like autoReconnect, maxReconnects etc. Format: `jdbc:mysql://{host}:{port}/{dbName}?{parameters}`"
+              + "1. The JDBC connection URL - which must contain the host, port and source db name and can optionally contain properties like autoReconnect, maxReconnects etc. Format: `jdbc:{mysql|postgresql}://{host}:{port}/{dbName}?{parameters}`"
```

Comment:
For PG we might need to have name-space too.
We have a field for the name in our SchemaReference. I am not sure, how it's getting populated with the current changes for PG.

Reply:
I think you mean schema in PG?

I would prefer if we didn't deal with those for now, because the PR is quite big already, but I will add a TODO